src/RL/representation/MIS/MISstaterepresentation.jl

```diff
@@ -0,0 +1,101 @@
+struct MISFeaturization <: AbstractFeaturization end
+
+"""
+    MISStateRepresentation{F, TS}
+
+This is a standard MIS representation using the graph of the problem with nodes having their assigned value as label.
+"""
+mutable struct MISStateRepresentation{F, TS} <: FeaturizedStateRepresentation{F, TS}
+    graph::LightGraphs.Graph
```

Comment:
It should have been great to use `Graphs.Graph` instead of `LightGraphs.Graph` since `LightGraphs` is deprecated and SeaPearl should transition towards `Graphs` in the long run.

I think that all the functions of `LightGraphs` used in this file are implemented in `Graphs` and that the work consists simply to replace `LightGraphs` with `Graphs`.